app/controllers/ChangeSuccessController.scala

```diff
@@ -80,17 +80,16 @@ class ChangeSuccessController @Inject()(vatSubscriptionService: VatSubscriptionS
           "Rendering InternalServerError")
         authComps.errorHandler.showInternalServerError
       } { title =>
-        val viewModel = ChangeSuccessViewModel(title, user.session.get(verifiedAgentEmail), entityName, None)
+        val viewModel = ChangeSuccessViewModel(title, user.session.get(verifiedAgentEmail), entityName)
         Ok(tradingNameSuccessView(viewModel, isRemoval))
       }
     }
 
   private[controllers] def renderBusinessNameView(implicit user: User[_]): Future[Result] =
     vatSubscriptionService.getCustomerInfo(user.vrn).map { vatSubscriptionResponse =>
       val entityName = vatSubscriptionResponse.fold(_ => None, details => details.entityName)
-      val contactPreference = vatSubscriptionResponse.fold(_ => None, details => details.contactPreference)
-      val title = "businessNameChangeSuccess.title"
-      val viewModel = ChangeSuccessViewModel(title, user.session.get(verifiedAgentEmail), entityName, contactPreference)
+      val title = "updateBusinessNameSuccess.title"
```

Comment:
No need to define the title message key here if there is no logic associated with it. I think the view can just call this message itself as you would typically do for other messages with no logic.